packages/scheduler/src/Timeline.ts

```diff
@@ -1,25 +1,29 @@
 /** @license MIT License (c) copyright 2010-2017 original author or authors */
 
 import { findIndex, removeAll } from '@most/prelude'
+import { Time, Timeline } from '@most/types'
+import ScheduledTaskImpl from './ScheduledTask'
```

Comment:
Is it possible to depend only on the `ScheduledTask` interface, rather than the `ScheduledTaskImpl` implementation?  Or does `TimelineImpl` need access to specific details of `ScheduledTaskimpl`?

Reply:
That's because of https://github.com/mostjs/core/pull/296#discussion_r325162673